styleguide/derek/_partials/solutions/leadSplit.ejs

```diff
@@ -0,0 +1,13 @@
+<!-- Lead pattern -->
+<!--
+.lead class naming not used in alignment with typical pattern structure
+instead here .lead-wrapper is used to avoid having to override bootstrap base class
+ -->
+<div class="lead-wrapper">
+  <div class="lead-headerWrapper">
+    <h1 class="lead-header left">Title goes here</h1>
```

Comment:
You shouldn't need to create the class after all. You can fix this by adjusting your layout to use the appropriate `subpanel` and `subpanel-inner` classes.